lib/Db/TagMapper.php

```diff
@@ -240,12 +210,59 @@ public function createDefaultTags(MailAccount $account): void {
 			}
 			$tags[] = $tag;
 		}
-		$dbTags = $this->getAllTagForUser($account->getUserId());
+		$dbTags = $this->getAllTagsForUser($account->getUserId());
 		$toInsert = array_udiff($tags, $dbTags, function (Tag $a, Tag $b) {
 			return strcmp($a->getImapLabel(), $b->getImapLabel());
 		});
 		foreach ($toInsert as $entity) {
 			$this->insert($entity);
 		}
 	}
+
+	public function deleteDuplicates(): void {
+		$qb = $this->db->getQueryBuilder();
+		$dqb = $this->db->getQueryBuilder();
```

Comment:
nitpick: I'd appreciate more descriptive variables like ``$deleteQueryBuilder`` or whatever the `d` stands for :)

and you can also move the initialization of the query builder down to where it's actually used. we don't need it before ~L240